### PR TITLE
Added validateInputs()

### PR DESCRIPTION
**Pull request by patelami3431**
Added validateInputs(), which checks the values of all the hyperparameters entered by the user or read from the file. It returns a vector of string which contains all the wrong inputs entered by the user or read from the file. It can be used to let user know what went wrong. I have included the prototype of the function in class as a comment for now but we can add it to the class later.

### CODE REVIEW

File: Network Class/Network.cpp [hunk 1]
```diff
@@ -503,6 +512,106 @@ Network::Network(const string& networkFilename, const string& validationDataFile
...
+std::vector<string> Network::validateInputs(string& str, const string& lr_range)
+{
+	bool result;
+	int go_ahead(0);
+	string temp;
+	int j(0);
+	//int end_of_file = filesize(trainingDataInfile);
+	std::vector<string> wrong_inputs;
+	std::vector<double> range = Strtok<double>(lr_range, " ");
+	while (!(result = ((learningRate > range[0]) && (learningRate < range[1]))))	//loop continues until the learning rate is between two given end points
```
Comment (kjhsdgf) [inline Network Class/Network.cpp]: why specify an upper bound for the learning rate? It may be unwise to use 3000 for eta, but that doesn't mean there will never be a scenario when it may be appropriate. As long as it's greater than 0, pretty much any real number will work.

**Review by kjhsdgf — APPROVED** [review]
Seems conceptually legit. Would you mind if I split the code up into 3 mutators for learningRate, epochs, and batchSize?

File: Network Class/Network.cpp (hunk 1 above)
Comment (patelami3431) [reply Network Class/Network.cpp]: Actually I am planning to have a setter method for range of learning Rate. So that the programmer can set a certain limits on the value of the learning rate.

Comment (patelami3431) [reply Network Class/Network.cpp]: Yeah definitely, it can be broken in 4 mutator methods: one for each hyperparameter that user enters or we receive from the file. And also, I was looking forward to have a mutator for setting up any directory for the user. So, user won't have to type in the complete location of the file while giving the training data file name or expected values file name. I can go ahead and code those mutators if you don't mind.

Comment (kjhsdgf) [reply Network Class/Network.cpp]: If you write it before I get around to it, then by all means.

Comment (patelami3431) [reply Network Class/Network.cpp]: Yeah I just updated the .cpp
